release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,421 @@
+
+
+[id="deprecated-and-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in earlier releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+For the most recent list of major functionality deprecated and removed, see the following table.
+Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+* GA: General Availability
+* TP: Technology Preview
+* DEP: Deprecated
+* REM: Removed
+* NA: Not applicable
+
+.Deprecated and removed features tracker
+[cols="5,1,1,1",options="header"]
+|===
+|Feature |{product-title-short} 4.3 |{product-title-short} 4.4 |{product-title-short} 4.5
+
+|`definitions.stackrox.io`
+|GA
+|DEP
+|DEP
+
+|`roxctl connectivity-map`
+|DEP
+|DEP
+|DEP
+
+|`roxctl generate netpol`
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/suppress` APIs
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/unsuppress` APIs
+|DEP
+|DEP
+|DEP
+
+|`/v1/cve/requests` APIs
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/suppress` APIs
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/unsuppress` APIs
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Management (1.0) menu item
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Report Creator permission
+|DEP
+|DEP
+|DEP
+
+|`/v1/availableAuthProviders` endpoint
+|GA
+|DEP
+|DEP
+
+|`/v1/tls-challenge` endpoint
+|GA
+|DEP
+|DEP
+
+|`/v1/summary/counts` endpoint
+|NA
+|NA
+|DEP
+
+|Reporting of Istio vulnerabilities
+|GA
+|DEP
+|DEP
+
+a|Custom Security Context Constraints (SCCs):
+
+* `stackrox-collector`
+* `stackrox-admission-control`
+* `stackrox-sensor`
+|DEP
+|REM
+|NA
+
+|CIS Docker v1.2.0 Compliance standard
+|DEP
+|REM
+|NA
+
+|PCI DSS 3.2.1 Compliance standard
+|DEP
+|REM
+|NA
+
+|NIST SP 800-53 Compliance standard
+|DEP
+|REM
+|NA
+
+|NIST SP 800-190 Compliance standard
+|DEP
+|REM
+|NA
+
+|HIPAA 164 Compliance standard
+|DEP
+|REM
+|NA
+
+|CIS Kubernetes v1.5 Compliance standard
+|DEP
+|REM
+|NA
+
+a|Reference image pull secret names for the Central components:
+
+* `stackrox`
+* `stackrox-scanner`
+|GA
+|REM
+|NA
+
+a|Reference image pull secret names for the secured cluster components:
+
+* `stackrox`
+* `stackrox-scanner`
+* `secured-cluster-services-main`
+* `secured-cluster-services-collector`
+* `collector-stackrox`
+|GA
+|REM
+|NA
+
+| `rhacs-collector*` and `rhacs-collector-slim*` images
+|NA
+|NA
+|DEP
+
+|Kernel support packages and driver download functionality
+|NA
+|NA
+|DEP
+
+|===
+
+[id="deprecated-features_{context}"]
+=== Deprecated features
+
+The following section provides information about deprecated features listed in the preceding table and other additional changes:
+
```

Comment:
Throughout this section, consider whether it's necessary to mention what Red Hat will do in a future release. IBM Style warns against making statements about the content of future releases. You made a general statement at the beginning of this section that deprecated items will be removed in a future release. With that in mind, I wonder if it would be reasonable in places where you state "XYZ has been deprecated and will be removed in a future release" to instead say "XYZ has been deprecated".